doc/api/events.markdown

```diff
@@ -339,7 +339,8 @@ Removes the specified `listener` from the listener array for the specified
 `removeListener` will remove, at most, one instance of a listener from the
 listener array. If any single listener has been added multiple times to the
 listener array for the specified `event`, then `removeListener` must be called
-multiple times to remove each instance.
+multiple times to remove each instance. Listeners removed from an event from
+within the emit cycle of that event will still be called in that emit cycle.  
```

Comment:
The term `emit cycle` isn't defined anywhere, is it? It might be better to say that it will run all listeners attached at the time an event is emitted. This is sort of tricky to explain just in text, and a code example would be extremely helpful.